packages/engine/src/scene/components/MediaComponent.ts

```diff
@@ -137,7 +137,7 @@ export const MediaComponent = defineComponent({
   schema: S.Object({
     controls: S.Bool(false),
     synchronize: S.Bool(true),
-    autoplay: S.Bool(true),
+    autoplay: S.Bool(false), //false = personal preference, this is super annoying when it just starts playing once added to a scene while editing
```

Comment:
ideally we should play but have it muted with an option to unmute all scene audio in the studio. ill bring it up with the design team